src/gui/AutomationPatternView.cpp

```diff
@@ -89,6 +96,22 @@ void AutomationPatternView::update()
 
 
 
+
+void AutomationPatternView::allowOpenToBar()
+{
+	if(Engine::getSong()->playMode() == 1)
+	{ 
+		TextFloat::displayMessage("Right Playmode, allow");
+	}//_cm.a.setEnabled(true); }
+	else
+	{
+		TextFloat::displayMessage("Disallow/Gray Out");
```

Comment:
If graying out "open to playhead" based on playMode is ok, this should disallow/gray out the option in context menus. Right now it just displays a message.